bindings/rust/wasmedge-sys/src/statistics.rs

```diff
@@ -79,10 +79,15 @@ impl Statistics {
     ///
     /// # Arguments
     ///
-    /// - `limit` specifies the cost limit.
+    /// * `limit` - The cost limit.
     pub fn set_cost_limit(&mut self, limit: u64) {
         unsafe { ffi::WasmEdge_StatisticsSetCostLimit(self.inner.0, limit) }
     }
+
+    /// Clears all data in this store.
```

Comment:
This is clear statistics, not all data in the store?

Reply:
Accepted. Thanks!